scripts/circle-ci/run-circle-tests.sh

```diff
@@ -13,15 +13,15 @@ function checkDocsBuild {
 
 CONTAINER_1=(':atlasdb-cassandra-integration-tests:check')
 
-CONTAINER_2=(':atlasdb-ete-tests:check' ':atlasdb-ete-test-utils:check')
+CONTAINER_2=(':atlasdb-ete-tests:check')
 
-CONTAINER_3=(':atlasdb-dbkvs:check' ':atlasdb-perf:postgresBenchmarkTest' ':atlasdb-cassandra:check')
+CONTAINER_3=(':atlasdb-dbkvs:check' ':atlasdb-cassandra:check')
 
-CONTAINER_4=(':atlasdb-cassandra-multinode-tests:check' ':atlasdb-impl-shared:check' ':atlasdb-cassandra-integration-tests:longTest' ':atlasdb-tests-shared:check' ':atlasdb-perf:check')
+CONTAINER_4=(':atlasdb-cassandra-multinode-tests:check' ':atlasdb-impl-shared:check' ':atlasdb-tests-shared:check' ':atlasdb-perf:check')
 
-CONTAINER_5=(':atlasdb-cassandra-integration-tests:memorySensitiveTest' ':atlasdb-ete-tests:longTest' ':lock-impl:check' ':atlasdb-dbkvs-tests:check')
+CONTAINER_5=(':lock-impl:check' ':atlasdb-dbkvs-tests:check' ':atlasdb-ete-test-utils:check' ':atlasdb-ete-tests:longTest')
 
-CONTAINER_6=(':atlasdb-ete-tests:startupIndependenceTest')
+CONTAINER_6=(':atlasdb-ete-tests:dbkvsTest' ':timelock-server:integTest')
```

Comment:
haha are our build times lower because we're excluding suites of tests or..?

Reply:
Startup independence was already refactored, so this task was not doing anything. We did remove the benchmark test run, but otherwise it's faster because we are not recreating KVSs and TMs 80+ times per test class anymore